hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteOperatorCoordinator.java

```diff
@@ -95,7 +96,7 @@ public class StreamWriteOperatorCoordinator
   /**
    * Hive config options.
    */
-  private final SerializableConfiguration hiveConf;
+  private final StorageConfiguration<org.apache.hadoop.conf.Configuration> storageConf;
```

Comment:
This is because hive has a conflict with Configuration?

Reply:
This is because the Flink classes import `org.apache.flink.configuration.Configuration`. So to reference Hadoop `Configuration` we need to use the fully-qualified class name.